roofit/roofitcore/inc/RooVectorDataStore.h

```diff
@@ -60,6 +60,25 @@ class RooVectorDataStore : public RooAbsDataStore {
 
   virtual ~RooVectorDataStore() ;
 
+  /// Output struct for the RooVectorDataStore::getArrays() helper function,
+  /// meant to be used for RooFit internal use.
+  struct ArraysStruct {
```

Comment:
Also move in "internal" doxygen section?

Reply:
Ok!